src/node_stat_watcher.h

```diff
@@ -26,7 +26,7 @@
 
 #include "node.h"
 #include "handle_wrap.h"
-#include "env.h"
+#include "env-inl.h"
```

Comment:
Sorry, I should have explained it more clearly last time. Compilation units (*.cc files) use env-inl.h but header files should normally use plain env.h because they only need the declarations, not inline definitions. My apologies!

Reply:
@bnoordhuis `node_stat_watcher.h` and `node_stat_watcher.cc` both import `env.h`

should this be (1) left as is or (2) removed from implementation file or (3) changed to `env-inl.h` in implementation file? sorry it's just so confusing to me 🤪

Reply:
Option (3) :-)

Reply:
Thank you and for your patience, very sorry about all the above.